lib/class-wp-theme-json-gutenberg.php

```diff
@@ -3480,4 +3480,39 @@ public function set_spacing_sizes() {
 
 		_wp_array_set( $this->theme_json, array( 'settings', 'spacing', 'spacingSizes', 'default' ), $spacing_sizes );
 	}
+
+	/**
+	 * Returns the CSS variable for a preset.
+	 *
+	 * @since 6.3.0
+	 *
+	 * @param array  $path Path to the preset.
+	 * @param string $slug Slug of the preset.
+	 * @return string CSS variable.
+	 */
+	public static function get_preset_css_var( $path, $slug ) {
+		$duotone_preset_metadata = static::get_preset_metadata_from_path( $path );
+		return static::replace_slug_in_string( $duotone_preset_metadata['css_vars'], $slug );
+	}
```

Comment:
I think we can get this working without adding this to the public API. The only place the `css_vars` for frontend duotone is being used is within the block supports now, and we already have it hardcoded a couple times there.

Reply:
That'd be lovely. For the record, I saw that this PR was reverted at https://github.com/WordPress/gutenberg/pull/49102

Reply:
This version of the PR did get reverted, and then was cleaned up and merged in this PR:  https://github.com/WordPress/gutenberg/pull/48995 

We were able to get the PR merged without adding anything to `lib/class-wp-theme-json-gutenberg.php`